gymnasium/utils/passive_env_checker.py

```diff
@@ -292,6 +292,34 @@ def _check_render_return(render_mode, render_return):
             logger.warn(
                 f"Depth-array rendering should return a numpy array with two axes, got {render_return.ndim}"
             )
+    elif render_mode == "rgbd_array":
```

Comment:
I don't think we need this code as I don't mind many other projects will use it

Reply:
good point. I'll remove it